examples/stable_diffusion_v2/ldm/models/diffusion/ddpm.py

```diff
@@ -352,9 +352,20 @@ def encode_first_stage(self, x):
     def get_first_stage_encoding(self, z):
         return self.scale_factor * z
 
-    def apply_model(self, x_noisy, t, c_concat=None, c_crossattn=None, return_ids=False, **kwargs):
-        x_recon = self.model(x_noisy, t, c_concat, c_crossattn, **kwargs)
-        return x_recon
+    def apply_model(self, x_noisy, t, cond, return_ids=False, **kwargs):
```

Comment:
I think we should preserve the explicit naming of `c_concat` and `c_crossattn`, referring to the discussion in https://github.com/mindspore-lab/mindone/pull/93#discussion_r1315332684

Reply:
As discussed, the `cond` now can accept either a dictionary or a Tensor. A docstring is added to make it clearer.